project/AppDependencies.scala

```diff
@@ -1,49 +1,38 @@
 import sbt._
 
-object FrontendBuild extends Build with MicroService {
-
-  val appName = "vat-registration-eligibility-frontend"
-
-  override lazy val appDependencies: Seq[ModuleID] = AppDependencies()
-}
-
 private object AppDependencies {
 
   import play.core.PlayVersion
   import play.sbt.PlayImport._
 
   private val playHealthVersion = "3.9.0-play-26"
-  private val logbackJsonLoggerVersion = "3.1.0"
-  private val govukTemplateVersion = "5.54.0-play-26"
+  private val logbackJsonLoggerVersion = "4.8.0"
+  private val govukTemplateVersion = "5.60.0-play-26"
   private val playUiVersion = "8.11.0-play-26"
```

Comment:
`play-ui` could be bumped up to `1.18.0`

Reply:
Play ui needs removing as it's no longer used but it's not a straightforward job removing the old assets - needs to be done as a larger piece of td

Reply:
:+1: